src/js/app.js

```diff
@@ -3,9 +3,9 @@
  *
  * This is where you write your app.
  */
-
-var UI = require('ui');
-var Vector2 = require('vector2');
+require('./main');
```

Comment:
Hmm, I would prefer main.js to continue to be the main entry, and that it requires the app within the app ready event. Is this possible?